packages/components/src/range-control/styles/range-control-styles.ts

```diff
@@ -130,6 +130,10 @@ export const Track = styled.span`
 	margin-top: ${ ( rangeHeightValue - railHeight ) / 2 }px;
 	top: 0;
 
+	@media not ( prefers-reduced-motion ) {
+		transition: width ease 0.1s;
+	}
```

Comment:
It looks like this animation is there to go hand in hand with the thumb's animation — we should remove it if we remove the thumb's animation.

(sidenote: animating the `width` is not very performant, and can cause the UI to feel sluggish instead of polishing the animation).

If were to keep the animation around, we could add a comment to help future readers understand that the track and thumb animations go hand in hand.